spec/github-pages/integration_spec.rb

```diff
@@ -0,0 +1,215 @@
+# frozen_string_literal: true
+
+RSpec.describe "Pages Gem Integration spec" do
+  def destination_file(file)
+    File.join tmp_dir, file
+  end
+
+  def source
+    @source ||= fixture_dir
+  end
+
+  def destination
+    @destination ||= tmp_dir
+  end
+
+  def env
+    { "BUNDLE_GEMFILE" => "#{source}/Gemfile" }
+  end
+
+  def run_or_raise(env, *cmd)
+    stdout_and_stderr_str, status = Open3.capture2e env, *cmd
+    raise StandardError, stdout_and_stderr_str if status.exitstatus != 0
+  end
+
+  let(:file) { "#{self.class.description}.html" }
```

Comment:
?

Reply:
This defaults the file to the context, meaning if you're in `jekyll-seo-tag` context, it'll look for `jekyll-seo-tag.html` by default.